src/Symfony/Component/Cache/composer.json

```diff
@@ -39,7 +39,7 @@
         "symfony/var-dumper": "^4.1.1|^5.0"
     },
     "conflict": {
-        "doctrine/dbal": "<2.5",
+        "doctrine/dbal": "<=2.5.0-RC2",
```

Comment:
DriverManager::getConnection(['url' =>]) has been introduced in 2.5.0 (does not exists in BETAs, nor RCs) doctrine/dbal@e8056f5

Reply:
`<2.5.1`? having a RC here is strange especially since we use prefer-stable so it should not be needed

Reply:
`2.5.0` is fine. `prefer-lowest` pick beta and rc version in test suite.

Reply:
can you revert? locally I can't reproduce?